packages/benchmarks/src/quizQuestions/bin/loadBadgeQuizQuestions.ts

```diff
@@ -2,35 +2,43 @@ import fs from "fs";
 import path from "path";
 import csv from "csv-parser";
 import { QuizQuestionData, QuizQuestionDataSchema } from "../QuizQuestionData";
-import { makeTags } from "./makeTags";
 
 const testDataPath = path.resolve(__dirname, "..", "..", "..", "testData");
 const csvFileInPath = path.resolve(testDataPath, "badge-questions.csv");
 const jsonFileOutPath = path.resolve(testDataPath, "badge-questions.json");
 
+const handleAnswers = (row: any) => {
+  const correctAnswers = row.Answer.trim()?.split("") || [];
+  const answers = ["A", "B", "C", "D", "E", "F"]
+    .map((label) => {
+      const isCorrect = correctAnswers.includes(label);
+      return {
+        answer: row[label],
+        isCorrect,
+        label,
+      };
+    })
+    .filter((answer) => answer.answer && answer.answer.trim() !== ""); // Remove empty answers
```

Comment:
i think this code could be a little cleaner. rather than filtering at the end, can you slice the array of ["A", "B", "C", ...] to the correct length before running the map over it?